client/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu
+
+RUN mkdir -p /srv/tunneldigger/client/
+
+COPY CMakeLists.txt /srv/tunneldigger/client/CMakeLists.txt
+COPY l2tp_client.c /srv/tunneldigger/client/l2tp_client.c
+COPY libasyncns /srv/tunneldigger/client/libasyncns
+
+RUN apt-get update && \
+    apt-get install -y \
+        cmake build-essential iproute2 pkg-config curl bison flex && \
+    mkdir -p /srv/libnl && \
+    cd /srv/libnl && \
+    curl -sL https://github.com/thom311/libnl/releases/download/libnl3_4_0/libnl-3.4.0.tar.gz \
```

Comment:
This is odd, I can build the client with just system packages on my Debain box. Is that not possible with Ubuntu?

I have installed `libnl-3-dev` and `libnl-genl-3-dev`, not sure which of them is needed.